src/doc/src/reference/manifest.md

```diff
@@ -511,6 +511,9 @@ package-name = "my-awesome-android-app"
 assets = "path/to/static"
 ```
 
+First-party `package.metadata` tables:
```

Comment:
Feel a bit abrupt we suddenly mention this. Not a good one but something like below?

```suggestion
Here is a list of `package.metadata` definitions maintained by the Rust Project:
```

Reply:
Did a bit of a hybrid.

> You'll need to look in the documentation for your tool to see how to use this field.
> For Rust Projects that use `package.metadata` tables, see:
> - [docs.rs](https://docs.rs/about/metadata)

Thoughts?

Reply:
Way better than mine!